server/jetstream_cluster.go

```diff
@@ -4450,7 +4450,10 @@ func (js *jetStream) processClusterCreateConsumer(ca *consumerAssignment, state
 			}
 		}
 
-		if rg.node == nil {
+		js.mu.RLock()
```

Comment:
We have a check above that it did not check. Best to capture node right as we enter the upper else clause.